src/utils/dates.js

```diff
@@ -10,3 +12,101 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
+export const handleNextPurchaseDate = (
```

Comment:
Typically naming something `handle` means that it plugs into some sort of on change listener, or it handles some sort of other process.

In this case, we're trying to update all the relevant information for an `Item` on purchase, so we should name it to reflect this. `updateItemOnPurchase` would illustrate what this helper does, and a description comment above can go into details on how `purchaseCount`, `dateNextPurchased` and `dateLastPurchased` need to be updated on a user marking an item as purchased.